docbook/reference/en/en-US/modules/Jaxb.xml

```diff
@@ -390,209 +376,6 @@ public class Book
     }
 }
 </programlisting>
-      </para>
-      <para>
-
-
-      </para>
-      <para>
-
-         This is how the JAXB Book class would be marshalled to JSON using the BadgerFish Convention:
-      </para>
-      <para>
-
-<programlisting>
-{"book":
-    {
-        "@title":"EJB 3.0",
-        "author":{"$":"Bill Burke"},
-        "ISBN":{"$":"596529260"}
-    }
-}
-</programlisting>
-      </para>
-      <para>
-
-         Notice that element values have a map associated with them and to get to the value of the element, you must
-         access the &quot;$&quot; variable. Here's an example of accessing the book in Javascript:
-      </para>
-      <para>
-
-<programlisting>
-var data = eval("(" + xhr.responseText + ")");
-document.getElementById("zone").innerHTML = data.book.@title;
-document.getElementById("zone").innerHTML += data.book.author.$;
-</programlisting>
-      </para>
-      <para>
-
-
-      </para>
-      <para>
-
-         To use the BadgerFish Convention you must use the
-         @org.jboss.resteasy.annotations.providers.jaxb.json.BadgerFish annotation on the JAXB class you are
-         marshalling/unmarshalling, or, on the JAX-RS resource method or parameter:
-      </para>
-      <para>
-
-<programlisting>
-@BadgerFish
-@XmlRootElement(name = "book")
-public class Book {...}
-</programlisting>
-      </para>
-      <para>
-
-         If you are returning a book on the JAX-RS method and you don't want to (or can't) pollute your JAXB classes
-         with RESTEasy annotations, add the annotation to the JAX-RS method:
-      </para>
-      <para>
-
-<programlisting>
-@BadgerFish
-@GET
-public Book getBook(...) {...}
-</programlisting>
-      </para>
-      <para>
-
-         If a Book is your input then you put it on the parameter:
-      </para>
-      <para>
-
-<programlisting>
-@POST
-public void newBook(@BadgerFish Book book) {...}
-</programlisting>
-      </para>
-      <para>
-
-
-      </para>
-      <para>
-
-         The default Jettison Mapped Convention would return JSON that looked like this:
-      </para>
-      <para>
-
-<programlisting>
-{ "book" :
-    {
-        "@title":"EJB 3.0",
-        "author":"Bill Burke",
-        "ISBN":596529260
-    }
-}
```

Comment:
Hey @liweinan,

Having removed the Jettison example, maybe you could include something like "The JSON output produced by Jackson would look like this:" with a Book in JSON format.

Reply:
yes sir!

Reply:
Lol.